peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -79,7 +79,7 @@ <h1 class="py-4">{{ document.title }}</h1>
 
                       {% block document-metadata-content-author %}
                         {% if document.author %}
-                          <dt>{% trans 'Author' %}</dt>
+                          <dt>{% trans document.author.model_label %}</dt>
```

Comment:
The label is already being translated by `_` elsewhere so it doesn't need to be double translated.